src/main/kotlin/com/team4099/robot2023/commands/drivetrain/DrivePathCommand.kt

```diff
@@ -420,16 +420,14 @@ private constructor(
       // Stop where we are if interrupted
       drivetrain.currentRequest =
         DrivetrainRequest.OpenLoop(
-          0.0.radians.perSecond, Pair(0.0.meters.perSecond, 0.0.meters.perSecond)
+          0.0.radians.perSecond,
+          Velocity2d.fromVelocityVectorToVelocity2d(0.0.meters.perSecond, 0.0.radians)
         )
     } else {
       // Execute one last time to end up in the final state of the trajectory
       // Since we weren't interrupted, we know curTime > endTime
       execute()
-      drivetrain.currentRequest =
-        DrivetrainRequest.OpenLoop(
-          0.0.radians.perSecond, Pair(0.0.meters.perSecond, 0.0.meters.perSecond)
-        )
+      drivetrain.currentRequest = DrivetrainRequest.OpenLoop(0.0.radians.perSecond, Velocity2d())
```

Comment:
how obvious is it here on first glance that an empty velocity2d constructor means 0 m/s for both? tbh, i want us to do what saraansh said and add ZERO singletons (just one instance w 0.0 m/s for both x and y) for these kind of dataclasses in the companion object.